src/Uno.UI.Runtime.Skia.Gtk/UI/Controls/UnoGtkWindow.cs

```diff
@@ -52,9 +52,6 @@ public UnoGtkWindow(WinUIWindow winUIWindow) : base(WindowType.Toplevel)
 
 		Host = new UnoGtkWindowHost(this, winUIWindow);
 
-		UpdateWindowPropertiesFromPackage();
```

Comment:
This was called from `UnoGtkWindow` constructor, but down the stack, it is expecting the window to be non-null, but it hasn't been yet created. So I moved these after the constructor finishes execution and `MainWindow` is assigned a non-null value.

Reply:
I reverted the `OnShown` change due to test failure.